audio.cpp

```diff
@@ -117,11 +120,16 @@ void ArduboyTunes::playNote(byte chan, byte note)
     return;
 
   // we only have frequencies for 128 notes
-  if (note > 127)
-    note = 127;
+  if (note > 127) {
+    return;
+  }
 
   timer_num = pgm_read_byte(tune_pin_to_timer_PGM + chan);
-  frequency2 = pgm_read_word(_midi_note_frequencies + note);
```

Comment:
That's why I starting putting the savings in the commit messages, so it's clear and no one would have to ask. :)